10-spatial-estimation.Rmd

```diff
@@ -1123,7 +1123,7 @@ leaflet() %>% addTiles() %>%
     title = "Predicted basal area")
 ```
 
-**Figure 10.20.** \ref{fig:figs} Predicted basal area over the entire Fort Saint Johns TSA using five nearest neighbor projected over the province of British Columbia.
+**Figure 10.20.** \ref{fig:figs} Predicted basal area over the entire Fort Saint Johns TSA using five nearest neighbor projected over the province of British Columbia. Data from City of Vancouver and licensed under the Open Government License - Vancouver. 
```

Comment:
I don't think this is data from the City of Van @pauldpickell is there another statement that belongs here?